report-generator/src/main/java/com/teamscale/report/testwise/model/factory/TestInfoFactory.java

```diff
@@ -65,11 +65,18 @@ public TestInfo createFor(TestCoverageBuilder testCoverageBuilder) {
 	public List<TestInfo> createTestInfosWithoutCoverage() {
 		ArrayList<TestInfo> results = new ArrayList<>();
 		for (TestDetails testDetails : testDetailsMap.values()) {
-			if (uniformPathsWithCoverage.contains(testDetails.uniformPath)) {
+			if (!processedTestUniformPaths.contains(testDetails.uniformPath)) {
 				TestInfoBuilder testInfo = new TestInfoBuilder(testDetails.uniformPath);
 				testInfo.setDetails(testDetails);
 				testInfo.setExecution(testExecutionsMap.get(testDetails.uniformPath));
 				results.add(testInfo.build());
+				processedTestUniformPaths.add(testDetails.uniformPath);
+			}
+		}
+		for (TestExecution testExecution : testExecutionsMap.values()) {
+			if (!processedTestUniformPaths.contains(testExecution.getUniformPath())) {
+				System.err.println("No coverage found found for " + testExecution.getUniformPath());
```

Comment:
Maybe we can add some more details, why we'd expect coverage here and what to do about it, e.g.
`"Test ${testExecution.getUniformPath()} was executed but no coverage file was found. Please make sure that xyz/do xyz"`
I'm not sure what the user might be able to do to fix this problem. If you don't either, we could just say "Please make sure that you configured the agent correctly." or something similar.